src/core_test/mpas_test_core_streams.F

```diff
@@ -35,7 +35,7 @@ module test_core_streams
    !>  that are created by this routine.
    !
    !-----------------------------------------------------------------------
-   subroutine test_core_streams_test(domain, threadErrs, ierr)
+   subroutine test_read_write_real_streams(domain, threadErrs, ierr)
```

Comment:
It would be good to update the routine name on line 22 in the comment block above.